static/coffee/services.coffee

```diff
@@ -490,7 +490,7 @@ services.factory('UserService', ['$http', '$httpParamSerializer', ($http, $httpP
     # Fetches non-partner users with optional activity statistics
     #----------------------------------------------------------------------------
     fetchNonPartner: (withActivity = false) ->
-      params = {non_partner: true, with_activity: withActivity}
+      params = {non_partner: false, with_activity: withActivity}
```

Comment:
`false` is the default value for `non_partner`, so we might as well use the `fetchAll` function here instead?